src/fixedValues.js

```diff
@@ -0,0 +1,5 @@
+const encounterTypeUuid ="4d911f5c-26fe-102b-80cb-0017a47871b2" ;
+const formName = "CURRENT PROGRAMME ADULTE VIH - RENDEZ-VOUS - Rwanda 2.1"  ;
+const sampleStatusUuid = "3ce163d8-26fe-102b-80cb-0017a47871b2";
+const referralStatusUuuid = "3cd93302-26fe-102b-80cb-0017a47871b2";
+const sampleDropOffUuid = "13662f27-9be5-4595-8bab-07b0e859d9f4";
```

Comment:
Let's call this file `constants.js`

Reply:
But actually this is only useful if these values are used in multiple files. Looks like they're only used in `ReportComponent.resource`. And actually these values are duplicated there. So let's just delete this file.

Reply:
Oops, sorry, just saw that these are also used in `ReportComponent`! Let's factor the common constants out into this `constants.js` file. You'll want to export each of those constants.